go-chaos/internal/dataloss_sim_helper_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Camunda Services GmbH
```

Comment:
🚀

Reply:
I guess that should be 2023 :sweat_smile: